packages/app/package.json

```diff
@@ -34,7 +34,6 @@
     "@vercel/webpack-asset-relocator-loader": "1.7.0",
     "css-loader": "^6.0.0",
     "electron": "16.0.6",
-    "eslint": "^7.32.0",
```

Comment:
@JohannesMerkt  I 've removed the eslint from package.json in app because we have eslint in the root package.json , I think that can be something which is synced across all packages.

Reply:
Yes it has not been used there anyways